packages/contracts/deploy/100_install_on_management_dao/100_install.ts

```diff
@@ -0,0 +1,315 @@
+import {VERSION, METADATA} from '../../plugin-settings';
```

Comment:
Just to double check, will `100_install.ts` be processed after a potential `20_something.ts` ?